client/stylesheets/_responsive.scss

```diff
@@ -235,6 +291,20 @@
 
 }
 
+@media (max-width: 750px) {
+    // .profileCardImg {
+    //     height: 138px !important;
+    // }
+
+    .managerOfferView {
+        margin-top: 15px !important;
+    }
+
+    .profileCardInfo li:first-child {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardInfo` should be written in lowercase with hyphens